src/main/java/teammates/ui/webapi/request/AccountCreateRequest.java

```diff
@@ -0,0 +1,46 @@
+package teammates.ui.webapi.request;
+
+import teammates.common.util.FieldValidator;
+
+/**
+ * The request of creating new account.
+ */
+public class AccountCreateRequest extends BasicRequest {
+
+    private String instructorEmail;
+    private String instructorName;
+    private String instructorInstitution;
+
+    public String getInstructorEmail() {
+        return instructorEmail;
+    }
+
+    public String getInstructorName() {
+        return this.instructorName;
+    }
+
+    public String getInstructorInstitution() {
+        return this.instructorInstitution;
+    }
+
+    public void setInstructorName(String name) {
+        this.instructorName = name;
+    }
+
+    public void setInstructorInstitution(String institution) {
+        this.instructorInstitution = institution;
+    }
+
+    public void setInstructorEmail(String instructorEmail) {
+        this.instructorEmail = instructorEmail;
+    }
+
+    @Override
+    public void validate() {
+        FieldValidator validator = new FieldValidator();
+        assertTrue(validator.getInvalidityInfoForPersonName(this.instructorName.trim()).isEmpty(), "validate name");
```

Comment:
The second parameter "validate name" is supposed to be the error message which is what `getInvalidityInfoForPersonName` returned?

Reply:
Updated, sorry, does not fully understand how validate works before